projects/hslayers/services/add-data/common-file.service.ts

```diff
@@ -408,7 +379,7 @@ export class HsAddDataCommonFileService extends HsAddDataCommonFileServiceParams
     try {
       this.loadingToLayman = true;
       if (!this.endpoint) {
-        this.pickEndpoint();
+        throw new Error('No endpoint available');
```

Comment:
Oddly enough, I have hit this error today, even though I am logged in to a Layman endpoint successfully. So the `pickEndpoint()` probably had some meaning here.

Reply:
b5d03f4 shall address that -- the global `endpoint` property was always `undefined`. Moving it into constructor was also not optimal, due to the timing of async code. So I have moved it into a local scope.